Makefile

```diff
@@ -265,12 +265,15 @@ ci-secret-bootstrap:
 
 ci-secret-generator:
 	$(CONTAINER_ENGINE) pull registry.svc.ci.openshift.org/ci/ci-secret-generator:latest
-	$(CONTAINER_ENGINE) run --rm -v "$(CURDIR)/core-services/ci-secret-generator/_config.yaml:/_config.yaml:z" -v "$(CURDIR)/core-services/ci-secret-bootstrap/_config.yaml:/_bootstrap_config.yaml:z" \
-		-v "$(kubeconfig_path):/_kubeconfig:z" \
-		-v "$(bw_password_path):/_bw_password:z" \
-		-e KUBECONFIG=_kubeconfig \
-		registry.svc.ci.openshift.org/ci/ci-secret-generator:latest \
-		--bw-password-path=/_bw_password --bw-user $(kerberos_id)@redhat.com --config=/_config.yaml --bootstrap-config=/_bootstrap_config.yaml --dry-run=$(dry_run)
+	@# This needs a bunch of stuff from the host for auth so just copy it there
+	$(eval ID = $(shell docker create registry.svc.ci.openshift.org/ci/ci-secret-generator))
+	$(CONTAINER_ENGINE) cp $(ID):/usr/bin/ci-secret-generator /tmp/secret-generator
```

Comment:
is it statically compiled? will this work for hongkai on a mac lol

Reply:
It won't work for hongkai but then again he is too afraid to run it in the first place so :shrug: